client/js/components/faq/DpFaqSupport.vue

```diff
@@ -0,0 +1,98 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <div
+    id="scFaq"
```

Comment:
why faq?

Reply:
and what is sc?

Reply:
and why its here named "support" and in the admin area "contact". Please be consistand with the wording

Reply:
it was the naming from the old code. I can remove it here.

Reply:
removed here https://github.com/demos-europe/demosplan-core/pull/2139/commits/f5b56648c2fd85d536152c3b08155412e7793529